### ISSUE TITLE
About the loss of SDE_3Dto2D

### ISSUE THREAD

**Issue by MorningEatDinner** [chao1224/MoleculeSDE#1]
I observed that in code **SDEModel3Dto2D_node_adj_dense_02**, loss is calculated by:
            losses_x = torch.square(score_x + z_x)  # [B, max_num_nodes, num_class_X] or [B, max_num_nodes, 1]
            losses_adj = torch.square(score_adj + z_adj)  # [B, max_num_nodes, max_num_nodes]

But in **SDE_model_2d_to_3d**, the code to calculate loss is
            loss_pos = torch.sum((scores - pos_noise) ** 2, -1)  # (num_node)

I'm confused why the 3D_to_2D code isn't      score - x.

**Comment by chao1224** [comment]
Hi @MorningEatDinner 
I am not sure what you mean by `why the 3D_to_2D code isn't score - x`. Can you explain this further?
